### PR TITLE
coalang: Add settings for IndentationBear

### PR DESCRIPTION
**Pull request by abhsag24**

### CODE REVIEW

File: coalib/bearlib/languages/definitions/c.coalang [hunk 1]
```diff
@@ -10,3 +10,4 @@ signed, sizeof, static, struct, switch, typedef, union, unsigned, void,
 volatile, while, \#include, \#define, \#undef, \#ifdef, \#ifndef,
 \#if, \#endif, \#else, \#elif, \#line, \#pragma
 special_chars = +-*\/.;\\,()[]{}\=<>|&^~?%!
+encapsulators = ( : ), [ : ]
```
Comment (sils1297) [inline coalib/bearlib/languages/definitions/c.coalang]: what should an encapsulator be please?

**Comment by sils1297** [comment]
https://github.com/coala-analyzer/coala/pull/2365/commits/e59bc2f83cc33a1faaa0f8de8477ed8269f35765 ack

**Comment by sils1297** [comment]
rebase and merge!

**Comment by abhsag24** [comment]
reack 18fa82a, 18fa82a

**Comment by abhsag24** [comment]
@rultor merge

**Comment by abhsag24** [comment]
reack 5875ab8

**Pull request merged**